package/yast2-kdump.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Nov  9 13:06:10 UTC 2015 - igonzalezsosa@suse.com
+
+- Fix AutoYaST profile schema (bsc#805275)
+- 3.1.20
```

Comment:
It seems that `3.1.20` has been already used in `master`, it should be `3.1.19.1` I guess...